extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,260 @@
+  ],
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Campaign ExperienceEvent Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Adobe Campaign extension to ExperienceEvent.",
+  "definitions": {
+    "experienceevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/message/id": {
+          "title": "Message Identifier",
+          "type": "number",
+          "description": "Adobe Campaign message's broadlog ID."
+        },
+        "https://ns.adobe.com/experience/campaign/message/profile": {
+          "title": "Profile",
+          "$ref": "https://ns.adobe.com/xdm/context/profile",
+          "description":
+            "The recipient of the message. This property is primarily used to link the message to a Profile using the EndCustomerId but it can also be used to freeze some properties of the profile at the time the message was sent."
+        },
+        "https://ns.adobe.com/experience/campaign/message/variant": {
+          "title": "Content Variant",
+          "type": "string",
+          "description":
+            "A campaign activity can have multiple variants (Multilingual or A/B testing) but a message delivered to a recipient always contains one of them.\n> IMPORTANT: In case of multilingual, the mirror page, the recipient can switch from one variant to another (ex: switch of the language)."
+        },
+        "https://ns.adobe.com/experience/campaign/message/seedMember": {
+          "title": "Seed Member",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "The recipient of this message is a member of a seed list for this campaign activity."
+        },
+        "https://ns.adobe.com/experience/campaign/message/quarantine": {
+          "title": "Quarantine",
+          "type": "boolean",
+          "default": false,
+          "description":
+            "Address moved to quarantine. The quarantine is set to true when the message failed to deliver in a way that indicates that no future messages will be deliverable either to this address."
+        },
+        "https://ns.adobe.com/experience/campaign/message/proofMember": {
+          "title": "Proof Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of the proof group for this campaign activity.",
+          "default": false
+        },
+        "https://ns.adobe.com/experience/campaign/message/controlGroupMember": {
+          "title": "Control Group Member",
+          "type": "boolean",
+          "description":
+            "The recipient of this message is a member of a control group for this campaign activity. Messages to control group members are not delivered.",
```

Comment:
Explain what a control group is. The term test group and control group seem to be used differently from other areas of the industry, where control group experiences are delivered (otherwise you would not be able to compare test performance)

Reply:
:+1:

Reply:
Control Group members are a random set of recipients which are not selected to be not targeted. Then the behaviour changes in the control group vs the targeted can be used to measure the campaign's effectiveness.

Reply:
please include these definitions in the description.

Reply:
👍